interfaces/cython/cantera/mixmaster/Mix.py

```diff
@@ -21,8 +21,8 @@ def __init__(self,g,name):
         self.name = name
         self.symbol = name
         self.index = g.species_index(name)
-        #self.minTemp = g.minTemp(self.index)
-        #self.maxTemp = g.maxTemp(self.index)
+        self.minTemp = g.min_temp
+        self.maxTemp = g.max_temp
```

Comment:
I think this should be the minimum / maximum temperatures for the specific species, not the phase (which can be different). This can be accessed as
`g.species(name).thermo.min_temp`.